.travis.yml

```diff
@@ -23,58 +22,35 @@ php:
   - 5.6
   - 7.0
 
-matrix:
-  fast_finish: true
-
 before_install:
-  - if [[ ${TRAVIS_PHP_VERSION:0:3} == "7.0" ]]; then sudo cp tests/php7-pool.conf ~/.phpenv/versions/$(phpenv version-name)/etc/php-fpm.d/; fi
-  - phpenv config-rm xdebug.ini
-  - mysql -u root -e 'create database prestashop;'
-  - cp ~/.phpenv/versions/$(phpenv version-name)/etc/php-fpm.conf.default ~/.phpenv/versions/$(phpenv version-name)/etc/php-fpm.conf
-  - sudo a2enmod rewrite actions fastcgi alias
-  - echo "cgi.fix_pathinfo = 1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
-  - echo "memory_limit = 512M" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
-  - echo "always_populate_raw_post_data = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
-  - ~/.phpenv/versions/$(phpenv version-name)/sbin/php-fpm
-  - sudo cp -f tests/travis-ci-apache-vhost /etc/apache2/sites-available/prestashop.conf
-  - sudo cp -Rf . /var/www/prestashop.unit.test
-  - sudo chmod 777 -R /var/www/
-  - sudo a2dissite 000-default
-  - sudo a2ensite prestashop.conf
```

Comment:
Instead od copy pasting the whole folder content into `/var/www`, we set the Apache vhost to run with the Travis working directory.